src/grt/test/pd3.tcl

```diff
@@ -5,7 +5,7 @@ read_def "gcd.def"
 
 set guide_file [make_result_file pd3.guide]
 
-set_routing_alpha 0.9 -min_fanout 10
+set_routing_alpha 0.9 -min_fanout 9
```

Comment:
Why is this changed?

Reply:
I've updated the check to be consistent with the option name (fanout = term_count -1). Too keep the ok files for this test, I've just updated the script.